### PR TITLE
Dev-2602: Update Text on new QT Launcher

### PR DESCRIPTION
**Pull request by danteruiz**
https://highfidelity.atlassian.net/browse/DEV-2602

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: launchers/qt/resources/qml/HFBase/Error.qml [hunk 1]
```diff
@@ -45,7 +45,7 @@ Item {
     HFTextRegular {
         id: description
 
-        text: "We seem to have a problem.\n   Please restart Launcher."
+        text: "We seem to have a problem.\n          Please restart."
```
Comment (zfox23) [inline launchers/qt/resources/qml/HFBase/Error.qml]: Are you using spaces to correctly position this text? What are you trying to do here?

Comment (zfox23) [reply launchers/qt/resources/qml/HFBase/Error.qml]: Please use [`HorizontalAlignment`](https://doc.qt.io/qt-5/qml-qtquick-text.html#horizontalAlignment-prop) instead.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22222/HighFidelity-Beta-PR16426-1a42ad0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22222/HighFidelity-Quest-Beta-PR16426-1a42ad0.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22222/HighFidelity-Beta-PR16426-1a42ad0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22222/HighFidelity-Beta-Interface-PR16426-1a42ad0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22223/HighFidelity-Beta-PR16426-d48097c.dmg

**Pull request merged**